build.cmd

```diff
@@ -76,6 +77,26 @@ echo.
 :: Eval the output from probe-win1.ps1
 for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy RemoteSigned "& ""%__SourceDir%\Native\probe-win.ps1"""') do %%a
 
+:GetDotNetCli
+
+set "__DotNetCliPath=%__RootBinDir%\tools\cli"
+if not "%__InstallCli%"=="" (
+    if exist "%__DotNetCliPath%" (rmdir /s /q "%__DotNetCliPath%")
+    if not %ErrorLevel%==0 (
+        echo "Could not remove current installation of dotnet CLI"
+        exit /b 1
+    )
+)
+if not exist "%__DotNetCliPath%" (
+    for /f "delims=" %%a in ('powershell -NoProfile -ExecutionPolicy RemoteSigned "& "%__SourceDir%\scripts\install-cli.ps1" -installdir "%__RootBinDir%\tools""') do (
+        echo "" > nul
+    )
+)
+if not exist "%__DotNetCliPath%" (
+    echo DotNet CLI could not be downloaded or installed.
```

Comment:
No, this is now needed to restore packages during managed build. I think it may be better to move it to after native compilation like in Unix.